deploy/csi-driver/030-node.yaml

```diff
@@ -54,7 +54,8 @@ spec:
               ovirt_url: $OVIRT_URL
               ovirt_username: $OVIRT_USERNAME
               ovirt_password: $OVIRT_PASSWORD
-              ovirt_cafile: $OVIRT_CAFILE
+              # set a valid path only if ca bundle has content
+              ovirt_cafile: ${OVIRT_CA_BUNDLE+$OVIRT_CAFILE}
```

Comment:
I think you wanted:

${OVIRT_CA_BUNDLE:+$OVIRT_CAFILE}

Reply:
you are right, it is safer